src/components/TextInput/_TextInput.scss

```diff
@@ -0,0 +1,75 @@
+$break-input-mobile: 400px;
+
+.textinput {
+  .inputfield {
+    @include space-inset-xs;
+    @include space-stack-xs;
+
```

Comment:
We should think about whether these belong in the component or if this should instead be determined by the parent container.

Reply:
@kaplanmichael 
`space-inset-xs` is the padding within the input field itself, which makes it part of the NYPL styling of the input field.  But, based on what you are suggesting, I will rethink how the other spacing is handled.

Reply:
Ah, OK, yes if the space is internal to the component (or field, in this case) then it does make sense. My bad.